demisto_sdk/commands/generate_docs/generate_integration_doc.py

```diff
@@ -220,6 +221,113 @@ def generate_setup_section(yaml_data: dict):
     return section
 
 
+# Incident Mirroring
+
+def trigger_generate_mirroring_section(yml_data: dict) -> bool:
+    """
+
+    Args:
+        yml_data: yml data of the integration.
+
+    Returns:
+        true if mirroring section should be generated.
+
+    """
+    script_data = yml_data.get('script', {})
+    sync_in = script_data.get('isremotesyncout', False)
+    sync_out = script_data.get('isremotesyncin', False)
+    return sync_out or sync_in
+
+
+def is_configuration_exists(yml_data: dict, names: list):
+    """
+    Args:
+        yml_data: yml data of the integration
+        names: list of configuration params to search for
+
+    Returns:
+        list of all configurations found.
+
+    """
+    confs = []
+    for conf in yml_data.get('configuration', []):
+        if conf.get('name', '') in names:
+            confs.append(conf)
+    return confs
+
+
+def generate_mirroring_section(yaml_data: dict) -> List[str]:
+    """
+
+    Args:
+        yaml_data: dict representing the yml file of the integration.
+
+    Returns: markdown section of Incident Mirroring.
+
+    """
+    integration_name = format(yaml_data['display'])
+    directions = {
+        'None': 'Turns off incident mirroring.',
+        'Incoming': f'Any changes in {integration_name} events (mirroring incoming fields) will be reflected in XSOAR incidents.',
```

Comment:
Why are you using "events" for the remote system incidents? I've seen you changed it on all places- but it could be other things (Jira ticket, CrowdStrike Falcon detection, etc).

Reply:
after talking with Dan, we decided to remove the %%incident type%% and use more general word, do you have any other suggestion? adding %% will also have us add special validation to validate that it has been removed by the developer.